scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -129,6 +129,10 @@ issues for {{product_name_short}}.
   <td><a href="/identity/docs/">{{cloudid_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/cloudidentity/cloudidentitymembership.md">CloudIdentityMembership</a></td>
 </tr>
+<tr>
+  <td><a href="/cloudids/docs/">{{cloudids_endpoint_name}}</a></td>
```

Comment:
This is a variable for service name, and the valid variable name is `{{ids_name}}`.